ui/raidboss/data/06-ew/dungeon/another_aloalo_island.ts

```diff
@@ -0,0 +1,1550 @@
+      type: 'GainsEffect',
+      netRegex: { effectId: 'E8B', source: 'Lala' },
+      run: (data, matches) => data.gainList.push(matches),
+    },
+    {
+      id: 'AAI Lala Planar Tactics',
+      type: 'Ability',
+      netRegex: { id: '8898', source: 'Lala', capture: false },
+      delaySeconds: 1.2,
+      durationSeconds: 10,
+      infoText: (data, _matches, output) => {
+        if (data.lalaSubtractive === undefined)
+          return;
+        return output.text!({ num: data.lalaSubtractive });
+      },
+      run: (data) => data.gainList = [],
+      outputStrings: {
+        text: {
+          en: '${num}',
+          ja: 'カウント: ${num}',
+        },
+      },
+    },
+    {
+      id: 'AAI Lala March',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['E83', 'E84', 'E85', 'E86'], source: 'Lala' },
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 7,
+      durationSeconds: 7,
+      alertText: (data, matches, output) => {
+        const map = {
+          'E83': 'front',
+          'E84': 'back',
+          'E85': 'left',
+          'E86': 'right',
+        }[matches.effectId];
+        if (map === undefined)
+          return;
+        if (data.lalaMyRotate === 'unknown')
+          return output[map]!();
+        if (isReverseRotate(data.lalaMyRotate, data.lalaMyTimes)) {
+          return {
+            'front': output.left!(),
+            'back': output.right!(),
+            'left': output.back!(),
+            'right': output.front!(),
+          }[map];
+        }
+        return {
+          'front': output.right!(),
+          'back': output.left!(),
+          'left': output.front!(),
+          'right': output.back!(),
+        }[map];
+      },
+      outputStrings: {
+        text: {
+          en: 'Mindhack',
+          de: 'Geistlenkung',
+          fr: 'Piratage mental', // FIXME
+          ja: '強制移動',
+          cn: '强制移动', // FIXME
+          ko: '강제이동', // FIXME
+        },
+        front: {
+          en: 'Mindhack: Forward',
+          de: 'Geistlenkung: Vorwärts',
+          fr: 'Piratage mental : Vers l\'avant',
+          ja: '強制移動 : 前',
+          cn: '强制移动 : 前',
+          ko: '강제이동: 앞',
+        },
+        back: {
+          en: 'Mindhack: Back',
+          de: 'Geistlenkung: Rückwärts',
+          fr: 'Piratage mental : Vers l\'arrière',
+          ja: '強制移動 : 後ろ',
+          cn: '强制移动 : 后',
+          ko: '강제이동: 뒤',
+        },
+        left: {
+          en: 'Mindhack: Left',
+          de: 'Geistlenkung: Links',
+          fr: 'Piratage mental : Vers la gauche',
+          ja: '強制移動 : 左',
+          cn: '强制移动 : 左',
+          ko: '강제이동: 왼쪽',
+        },
+        right: {
+          en: 'Mindhack: Right',
+          de: 'Geistlenkung: Rechts',
+          fr: 'Piratage mental : Vers la droite',
+          ja: '強制移動 : 右',
+          cn: '强制移动 : 右',
+          ko: '강제이동: 오른쪽',
+        },
+      },
```

Comment:
I think using these Mindhack callouts for Planar Tactics is confusing; I would instead use a simple "Face Left/Right" call.